container_registry/container_analysis/README.md

```diff
@@ -0,0 +1,59 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
+Cloud Platform logo" title="Google Cloud Platform" align="right" height="96"
+width="96"/>
+
+# Google Cloud Container Analysis Samples
+
+
+Container Analysis scans container images stored in Container Registry for vulnerabilities.
+Continuous automated analysis of containers keep you informed about known vulnerabilities so 
+that you can review and address issues before deployment.
+
+Additionally, third-party metadata providers can use Container Analysis to store and 
+retrieve additional metadata for their customers' images, such as packages installed in an image.
+
+
+## Description
+
+These samples show how to use the [Google Cloud Container Analysis Client Library](https://cloud.google.com/container-registry/docs/reference/libraries).
+
+## Build and Run
+1.  **Enable APIs** - [Enable the Container Analysis API](https://console.cloud.google.com/flows/enableapi?apiid=containeranalysis.googleapis.com)
+    and create a new project or select an existing project.
+1.  **Install and Initialize Cloud SDK**
+    Follow instructions from the available [quickstarts](https://cloud.google.com/sdk/docs/quickstarts)
+1.  **Clone the repo** and cd into this directory
+
+```
+    $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples
+    $ cd python-docs-samples/container-registry/container-analysis
+```
+
+1. **Install Dependencies**
+
+```
+    $ python3 -m venv ./venv
+    $ source ./venv/bin/activate
+    $ pip install -r requirements.txt
```

Comment:
Got two errors installing dependencies (note, I did fetch the upstream and rebase your work on top before I tried anything)

`grafeas 0.1.0 has requirement google-api-core[grpc]<2.0.0dev,>=1.6.0, but you'll have google-api-core 0.1.4 which is incompatible.
google-cloud-containeranalysis 0.1.0 has requirement google-api-core[grpc]<2.0.0dev,>=1.4.1, but you'll have google-api-core 0.1.4 which is incompatible.`

Reply:
Hmm I haven't been able to reproduce this. And you're using a virtualenv?

It's strange because I'm seeing google_api_core-0.1.4 in my site-packages, but I didn't get any error messages

Reply:
ok, actually I updated the pubsub version, and I think everything should be ok now. (that updated google_api_core to be in the right range)